### ISSUE TITLE
New interface does not support `None` in pytrees 

### ISSUE THREAD

**Issue by PhilipVinc** [google/orbax#721]
None is registered as a pytree with jax, so it can be found inside pytrees. However this code ~crashes~ prints a deprecation warning.

```python
import orbax.checkpoint as ocp

pytree = {'a': 0, 'b':None}
options = ocp.CheckpointManagerOptions()
mngr = ocp.CheckpointManager(
  ocp.test_utils.create_empty('/tmp/ckpt22/'),
  options=options,
)

mngr.save(0, args=ocp.args.StandardSave(pytree))
```

**Comment by cpgaffney1** [comment]
It works when I run it. Are you updated to the latest version? I think we submitted a recent fix related to this issue.

**Comment by PhilipVinc** [comment]
I just installed `orbax-checkpoint` from master, and running the script above prints the deprecation warning

```python
WARNING:absl:SaveArgs.aggregate is deprecated, please use custom TypeHandler (https://orbax.readthedocs.io/en/latest/custom_handlers.html#typehandler) or contact Orbax team to migrate before May 1st, 2024.
```

So I would say it's not supported by the new interface?

**Comment by PhilipVinc** [comment]
Ah sorry @cpgaffney1 , in order to address the deprecation warnings in my code I had made them crash and forgot about it.

So it does not crash, but you warn it to be deprecated and it's unclear to me how to address this warning.

I would assume that if an object is declared to be a pytree with jax, it should 'just work' ?

**Comment by cpgaffney1** [comment]
Ah ok, so this seems to be a dup of your other issue then? https://github.com/google/orbax/issues/720.

Sorry about the misleading warning, but don't worry about it. This message is just to discourage people from explicitly setting the `aggregate` option, which you're not doing. I'll see about making sure that message doesn't get printed in unrelated scenarios.

**Comment by niketkumar** [comment]
A fix has been pushed. Should be available in release `0.5.7`. Please let us know if the issue persists.
